src/test/resources/api_query_response.json

```diff
@@ -0,0 +1,116 @@
+[ {
+  "cristin_project_id" : "2040291",
+  "main_language" : "en",
+  "titles" : [ {
+    "title" : "Opportunities and Challenges for Integrating Sámi Reindeer Herding Traditional Environmental Knowledge in Environmental Governance",
+    "language" : "en"
+  } ],
+  "participants" : [ {
+    "cristinPersonId" : "327970",
+    "fullName" : "Turi, Ellen Inga"
```

Comment:
Is this a synthetic example? We should not be publishing personal information without the consent of the person in question (github is viewable by anyone).